### PR TITLE
Ea/superset oauth

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -168,6 +196,20 @@ services:
       - 9001:9001 # enable for UI access
     command: server --certs-dir /certs /data
 
+  superset_init:
+    environment:
+      PGPASSWORD: postgres
+      PGUSER: postgres
+      PGHOST: db
+    image: postgis/postgis:12-3.1-alpine
```
Comment (yakloinsteak) [inline docker/docker-compose.yml]: Maybe add a comment for those that need to override this. Or, I can take a pass at a universal postgis Dockerfile (which worked for Kevin and me). Don't want to derail the main goal here, though.

File: superset/op/superset_config_docker.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#
```
Comment (yakloinsteak) [inline superset/op/superset_config_docker.py]: I didn't dig much, but do we need all three superset config python files? I'm not sure if one or more of them might be trash.

**Review by yakloinsteak — COMMENTED** [review]
This looks great.

File: docker/docker-compose.yml (hunk 1 above)
Comment (eanders) [reply docker/docker-compose.yml]: Thanks!  I'll add a comment later, but for anyone looking now, you'll need to override the posts for superset like so: 
```
superset_init:
    image: gangstead/postgis:13-3.1-arm
```

File: superset/op/superset_config_docker.py (hunk 2 above)
Comment (eanders) [reply superset/op/superset_config_docker.py]: I believe one can go, it was a failed attempt.  The other two can probably get merged, I haven't figured out what the correct setup should be yet.

**Pull request merged**